frontend/packages/rhoas-plugin/src/models/rhoas.ts

```diff
@@ -0,0 +1,41 @@
+import { K8sKind } from '@console/internal/module/k8s';
+import { API_GROUP, API_VERSION } from '../const';
+
+export const CloudServicesRequestModel: K8sKind = {
+  apiGroup: API_GROUP,
+  apiVersion: API_VERSION,
+  kind: 'CloudServicesRequest',
+  id: 'cloudservicesrequest',
+  plural: 'cloudservicesrequests',
+  label: 'Cloud Services Request',
+  labelPlural: 'Cloud Services Requests',
+  abbr: 'CSCR',
+  namespaced: true,
+  crd: true,
+};
+
+export const KafkaConnectionModel: K8sKind = {
+  apiGroup: API_GROUP,
+  apiVersion: API_VERSION,
+  kind: 'KafkaConnection',
+  id: 'kafkaconnection',
+  plural: 'kafkaconnections',
+  label: 'Kafka Connection',
+  labelPlural: 'Kafka Connections',
+  abbr: 'AKC',
```

Comment:
why `AKC` , shouldn't this be `KC`

Reply:
KC is used for cluster defitions, keycloak and couple other reaources. Conflicts were almost guaranteed so we added Apache to it